src/main/webapp/app/entities/event/event-detail.tsx

```diff
@@ -1,16 +1,20 @@
+import './event-details.scss';
+
 import React from 'react';
 import { connect } from 'react-redux';
 import { Link, RouteComponentProps } from 'react-router-dom';
-import { Button, Row, Col } from 'reactstrap';
+import { Button } from 'reactstrap';
 // tslint:disable-next-line:no-unused-variable
-import { Translate, ICrudGetAction, byteSize, TextFormat } from 'react-jhipster';
+import { Translate, TextFormat } from 'react-jhipster';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 
 import { IRootState } from 'app/shared/reducers';
 import { getEntity } from './event.reducer';
-import { IEvent } from 'app/shared/model/event.model';
+import { faCalendarAlt, faClock, faMapMarkerAlt, faMoneyBillAlt } from '@fortawesome/free-solid-svg-icons';
```

Comment:
this one should be import in [icon-loader.ts](https://github.com/Five-Fishes/Club-Management-React-Client/blob/dev-version-1/src/main/webapp/app/config/icon-loader.ts)